resources.repos

```diff
@@ -0,0 +1,17 @@
+repositories:
+  rosdistro:
+    type: git
+    url: git@github.com:ros2/rosdistro.git
+    version: ros2
+  rosdep:
+    type: git
+    url: git@github.com:ros/rosdistro.git
+    version: master
+  rosforks:
+    type: git
+    url: git@github.com:rosindex/rosforks.git
+    version: master
+  deploy:
+    type: git
+    url: git@github.com:rosindex/rosindex.github.io.git
```

Comment:
`ros-infrastructure/rosindex`?

Reply:
+1 I don't think we'd want to deploy against public rosindex, though I can't seem to find said repository in `ros-infrastructure` :/